src/main.c

```diff
@@ -192,6 +190,60 @@ void kill_childs()
 	debug(D_EXIT, "All threads/childs stopped.");
 }
 
+struct option_def options[] = {
+	// opt description                                                       arg name                     default value
+	{'c', "Load alternate configuration file",                               "config_file",                          CONFIG_DIR "/" CONFIG_FILENAME},
+	{'D', "Disable fork into background",                                    NULL,                                   NULL},
+	{'h', "Display help message",                                            NULL,                                   NULL},
+	{'P', "File to save a pid while running",                                "FILE",                                 NULL},
+	{'p', "Port to listen. Can be from 1 to 65535.",                         "port_number",                          "19999"},
+	{'s', "Path to access host /proc and /sys when running in a container.", "PATH",                                 NULL},
+	{'t', "The frequency in seconds, for data collection. \
+Same as 'update every' config file option.",                                 "seconds",                              "1"},
+	{'l', "The number of entries the netdata daemon will keep in memory \
+for each chart dimension. Same as 'history config file option.",             "lines_to_save",                        NULL},
```

Comment:
I think this is the `history` option.
It is probably best to rename it.

Why this had this name: The first test I made 3 years ago was a daemon that was generating static JSON files in tmpfs that were served by apache/nginx, so this was `lines to save`. It is a left over from these early days.